src/pages/Lending/YourLoans.tsx

```diff
@@ -1,23 +1,125 @@
-import { Button, type GridProps, SimpleGrid, Stack } from '@chakra-ui/react'
-import { btcAssetId } from '@shapeshiftoss/caip'
-import { useMemo } from 'react'
+import { Button, type GridProps, SimpleGrid, Skeleton, Stack } from '@chakra-ui/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { fromAssetId } from '@shapeshiftoss/caip'
+import { useCallback, useMemo } from 'react'
 import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router'
 import { Amount } from 'components/Amount/Amount'
 import { HelperTooltip } from 'components/HelperTooltip/HelperTooltip'
 import { Main } from 'components/Layout/Main'
 import { AssetCell } from 'components/StakingVaults/Cells'
 import { RawText, Text } from 'components/Text'
+import type { Asset } from 'lib/asset-service'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { selectFirstAccountIdByChainId } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
 
 import { LendingHeader } from './components/LendingHeader'
+import { useLendingPositionData } from './hooks/useLendingPositionData'
+import { useLendingSupportedAssets } from './hooks/useLendingSupportedAssets'
 
 export const lendingRowGrid: GridProps['gridTemplateColumns'] = {
   base: 'minmax(150px, 1fr) repeat(1, minmax(40px, max-content))',
   md: 'repeat(4, 1fr)',
 }
 
+type LendingRowGridProps = {
+  asset: Asset
+  onPoolClick: (assetId: AssetId) => void
+}
+
+const LendingRowGrid = ({ asset, onPoolClick }: LendingRowGridProps) => {
+  // TODO(gomes): this only handles displaying positions of account 0 for now - we may want to make this component accomodate positions over all accounts
+  // however, this would rug the "Repayment Lock" data, so we need to figure out a UX way around this
```

Comment:
@reallybeard FYI - same UX issue here as the repayment lock one I was mentioning in `/pool/:pool` route